src/cascadia/TerminalApp/AppActionHandlers.cpp

```diff
@@ -39,7 +39,7 @@ namespace winrt::TerminalApp::implementation
     void TerminalPage::_HandleDuplicateTab(const IInspectable& /*sender*/,
                                            const ActionEventArgs& args)
     {
-        _DuplicateTabViewItem();
+        _DuplicateFocusedTab();

```

Comment:
A stupid question: if a tab is not focused, can it be duplicated?

Reply:
I kinda want see a image of gif demonstrating this feature. There's no image so I can only guess how it will behave based on the method name 😅

Reply:
Here I just renamed the method for duplicating the current tab (in the `TerminalPage` terminology, current tab is called focused tab, since it is returned by GetFocusedTabIndex).

The addition in the rest of the code is to allow duplication of the tab which is not the current tab, by clicking on "Duplicate Tab" in the TabView's context menu.